src/appleseed/foundation/meta/tests/test_exrimagefilewriter.cpp

```diff
@@ -64,8 +64,11 @@ TEST_SUITE(Foundation_Image_EXRImageFileWriter)
         attrs.insert("appleseed:test:IntAttr", 32);
         attrs.insert("appleseed:test:UnsignedIntAttr", static_cast<size_t>(32));
 
-        EXRImageFileWriter writer;
-        writer.write(Filename, image, attrs);
+        GenericImageFileWriter writer(Filename);
```

Comment:
Actually the test suite name and the file name must be updated since this is now testing the `GenericImageFileWriter` class.

Reply:
Done